exchange/utils.go

```diff
@@ -226,21 +233,33 @@ func getAuctionBidderRequests(req AuctionRequest,
 		}
 	}
 
-	reqExt, err := getExtJson(req.BidRequest, requestExt)
-	if err != nil {
-		return nil, []error{err}
-	}
-
 	var errs []error
 	for bidder, imps := range impsByBidder {
 		coreBidder := resolveBidder(bidder, aliases)
 
 		reqCopy := *req.BidRequest
 		reqCopy.Imp = imps
-		reqCopy.Ext = reqExt
 
 		prepareSource(&reqCopy, bidder, sChainsByBidder)
 
+		if len(bidderExt) != 0 {
+			bidderName := openrtb_ext.BidderName(bidder)
```

Comment:
Why do you cast the string `bidder` into a `bidderName`, just to cast it back into a string?

Reply:
Fixed it.